pkgs/tools/graphics/ploticus/default.nix

```diff
@@ -44,6 +43,8 @@ stdenv.mkDerivation rec {
 
   preBuild = ''
     cd src
+    # Set the location of the PREFABS directory.
```

Comment:
Ugh, looks like I forgot to add the `postPatch` section for the patch to work. Example: https://github.com/NixOS/nixpkgs/blob/c954a09fb832dab15dff73465a64136625876912/pkgs/applications/misc/udiskie/default.nix#L32-L34

Reply:
Should I switch back to patch + substitute, or is the `sed` ok?

Reply:
It is probably better since sed does not report failure to match.